packages/schematics/angular/application/files/browserslist.template

```diff
@@ -1,11 +1,16 @@
-# This file is currently used by autoprefixer to adjust CSS to support the below specified browsers
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
 # For additional information regarding the format and rule options, please see:
 # https://github.com/browserslist/browserslist#queries
-#
-# For IE 9-11 support, please remove 'not' from the last line of the file and adjust as needed
+
+# Googlebot uses an older version of Chrome
+# For additional information see: https://developers.google.com/search/docs/guides/rendering
+
+# For Googlebot support, remove '#' next to 'Chrome 41'.
```

Comment:
why aren't these comments adjacent to the line you're supposed to change? because if the file gets long then not all instructions are on the first screen?

Reply:
Good point